tests/test_skyWcs.py

```diff
@@ -304,6 +304,19 @@ def safeOffset(crval, direction, amount):
 
         return wcs
 
+    def checkNonFitsWcs(self, wcs):
+        """Check SkyWcs.getFitsMetadata for a WCS that cannot be represented as a FITS-WCS
+        """
+        # the modified WCS should not be representable as pure FITS-WCS
+        self.assertFalse(wcs.isFits)
+        with self.assertRaises(RuntimeError):
+            wcs.getFitsMetadata(True)
+
+        # the approximation returned by getFitsMetadata is poor (pure TAN) until DM-13170
+        wcsFromMetadata = makeSkyWcs(wcs.getFitsMetadata(False))
+        bbox = Box2I(Point2I(0, 0), Extent2I(2000, 2000))  # arbitrary but reasonable
```

Comment:
elsewhere you have changed the (0,0) point to something else.  Do you want to use that here as well?